lib/src/github_sign_in_page.dart

```diff
@@ -35,18 +35,20 @@ class _GitHubSignInPageState extends State<GitHubSignInPage> {
     return Scaffold(
         appBar: new AppBar(
           title: Text(widget.title),
+          centerTitle: true,
```

Comment:
I believe not all of people like title placed at center

Reply:
I can add this as a parameter if you want

Reply:
New parameter added ;)

Reply:
I meant the `centerTitle` I don't know if all of people like it or not.

Reply:
It's an optional parameter, people can use it or not ^_^